Homework_16/Task_1.py

```diff
@@ -0,0 +1,6 @@
+temp = [22, 25, 19, 23, 25, 26, 23]
+temp_sum = 0
+for c in temp:
+    temp_sum += c
+    avg = temp_sum//len(temp)
```

Comment:
avg-ს ყოველ ჯერზე დათვლა არაა საჭირო sum-ს რო დაითვლი ბოლოს for-ის გარეთ ერთხელ შეგიძლია avg დათვალო